config/jobs/kubernetes/generated/generated.yaml

```
@@ -1763,6 +1775,9 @@ periodics:
       - --test_args=--ginkgo.focus=\[Feature:Ingress\] --ginkgo.skip=\[Unreleased\] --minStartupPods=8
       env:
       image: gcr.io/k8s-testimages/kubekins-e2e:v20190510-1285909-master
+  annotations:
+    testgrid-tab-name: gce-cos-k8sstable3-ingress
+    testgrid-dashboards: sig-release-1.11-all, sig-release-1.11-blocking
```

Not going to point them out exhaustively here since this is generated, just pointing out that there are some here
```suggestion
    testgrid-dashboards: sig-release-1.11-blocking, sig-release-1.11-all
```